src/main/webapp/pages/projects/settings/pages/remote.html

```diff
@@ -1,69 +1,35 @@
-            }
-        };
-        </script>
-</head>
-<body>
-<div layout:fragment="settings-content" th:with="status = ${project.getRemoteStatus()}">
-    <h2 class="h2-muted" th:text="#{project.settings.sync.heading}"></h2>
-    <div class="form-group">
-        <label class="control-label" th:text="#{project.settings.sync.lastSync}">_Last Sync_</label>
-        <div>
-            <span th:text="${#calendars.format(project.getRemoteStatus().getLastUpdate(), 'dd MMM yyyy hh:mma')}"></span>
-            <button id="forceSync" role="button" class="btn btn-default btn-sm"
-                    th:text="#{project.settings.sync.forceSync}">_Force Sync_
-            </button>
-        </div>
-    </div>
-    <div class="form-group connection-wrapper">
-        <label class="control-label" th:text="#{project.settings.api}">_API_</label>
-        <div>
-            <a href="#" th:href="@{/remote_api/{id}(id=${status.getApi().getId()})}"
-               th:text="${status.getApi().getLabel()}">_api_</a>
-            <span id="api-status" class="api-status js-status-wrapper">
-                <i class="fa fa-spinner fa-spin"></i>
-            </span>
+            window.PAGE = {
+                _USER: /*[[${session.CURRENT_USER_DETAILS}]]*/ {}
+            };
 
-            <button type="button" class="oauth-connect-link btn btn-default hidden js-connect-btn" id="connect-button"
-                    data-toggle="modal" data-target="#remote-connect-wrapper"
-                    th:text="#{remoteapi.status.connect.button}" data:api-id="${status.getApi().getId()}">
-                _Connect_
-            </button>
-        </div>
-    </div>
-    <div class="form-group">
-        <label class="control-label" for="frequency" th:text="#{project.settings.sync.frequency}">_Sync
-            Frequency_</label>
-        <select id="frequency" class="form-control input-full sync-setting">
-            <option th:each="freq : ${frequencies}" th:value="${freq}"
-                    th:text="#{'project.settings.frequency.' + ${freq}}"
-                    th:selected="${project.getSyncFrequency() == freq}"></option>
-        </select>
-    </div>
-    <div class="form-group">
-        <label class="control-label" for="frequency" th:text="#{project.settings.sync.user}">_Sync User_</label>
-        <div>
-            <a href="#" th:text="${status.getReadBy().getLabel()}"
-               th:href="@{/users/{id}(id=${status.getReadBy().getId()})}">_User</a>
-            <span class="api-connected-action" hidden="hidden">
-              <button id="becomeSyncUser" role="button" class="btn btn-default btn-sm"
-                      th:text="#{project.settings.sync.becomeSyncUser}">_Become owner_</button>
-            </span>
-        </div>
+            window.translations = [{
+                "ProjectRemoteSettings.frequency.NEVER": /*[[#{ProjectRemoteSettings.frequency.NEVER}]]*/ "",
```

Comment:
I have translation for these in my branch.  Why are you adding them to the page?

Reply:
I had this part done before I saw that you had them in your branch :)

Reply:
Removed in de55c629770a6fc3924ffbd65eb135a6082c68e8